src/pages/home/report/ReportActionItem.js

```diff
@@ -118,6 +119,24 @@ class ReportActionItem extends Component {
         this.setState({isContextMenuActive: ReportActionContextMenu.isActiveReportAction(this.props.action.reportActionID)});
     }
 
+    renderWrappedComponents(children) {
```

Comment:
@mollfpr looks good 🎉
Please write JSdoc for the func. And let's rename the function so that it better describes what the function does?

wrapWithPressable??

Reply:
Yeah, that’s sounds better 👍